doc/examples/mnist_mlp/CMakeLists.txt

```diff
@@ -18,12 +18,9 @@ add_executable(mnist_mlp mnist_loader.cpp mnist_mlp.cpp)
 add_dependencies(mnist_mlp ngraph cpu_backend)
 target_link_libraries(mnist_mlp ngraph cpu_backend)
 if (NGRAPH_DISTRIBUTED_ENABLE)
-    find_package(MPI REQUIRED)
     add_definitions(-DNGRAPH_DISTRIBUTED)
-    include_directories(SYSTEM ${MPI_C_INCLUDE_PATH} ${MPI_CXX_INCLUDE_PATH})
-    link_directories(${MPI_C_LIBRARIES} ${MPI_CXX_LIBRARIES})
-    link_libraries(${MPI_CXX_LIBRARIES})
     add_executable(dist_mnist_mlp mnist_loader.cpp dist_mnist_mlp.cpp)
-    add_dependencies(dist_mnist_mlp ngraph cpu_backend)
-    target_link_libraries(dist_mnist_mlp ngraph cpu_backend)
+    add_dependencies(dist_mnist_mlp ngraph cpu_backend libmlsl)
```

Comment:
This is not needed. The target_link_libraries automatically adds dependencies for libraries linked in.